go-update/src/main/java/org/reactome/release/goupdate/DuplicateCleaner.java

```diff
@@ -0,0 +1,215 @@
+package org.reactome.release.goupdate;
+
+import java.io.FileInputStream;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.GKSchemaAttribute;
+import org.gk.schema.InvalidAttributeException;
+import org.reactome.release.common.ReleaseStep;
+
+/**
+ * Stand-alone class with main method that can be used to clean up duplicate GO accessions.
+ * @author sshorser
+ *
+ */
+public class DuplicateCleaner extends ReleaseStep
+{
+	private static final Logger logger = LogManager.getLogger();
+	private MySQLAdaptor adaptor;
+	private static final String[] goClasses = {ReactomeJavaConstants.GO_BiologicalProcess, ReactomeJavaConstants.GO_MolecularFunction, ReactomeJavaConstants.GO_CellularComponent, ReactomeJavaConstants.Compartment};
+
+	public static void main(String[] args)
+	{
+		String pathToResources = "src/main/resources/go-update.properties";
```

Comment:
Should this be configed?

Reply:
If the user species a path to the properties file on the command line, that is used. This line here sets a default value that only gets used when no argument is passed in.